test/ctia/task/migration/migrate_es_stores_test.clj

```diff
@@ -521,6 +521,7 @@
                 "feedback - finished migrating 0 documents"
                 "casebook - finished migrating 100 documents"
                 "sighting - finished migrating 100 documents"
+                "identity-assertion - finished migrating 0 documents"
```

Comment:
you need to test the migration of your entity, import some fixtures like the other entities